libcrux-ml-kem/hax.py

```diff
@@ -30,36 +30,6 @@ def shell(command, expect=0, cwd=None, env={}):
 class extractAction(argparse.Action):
 
     def __call__(self, parser, args, values, option_string=None) -> None:
-        # Extract sha3 interfaces
```

Comment:
In the F* extracted from this crate, we no longer rely on any SHA-3 extracted interfaces. This is because we only extract and check the _interface_ of `hash_functions.rs` not its implementation. I have added a comment to this effect to the top of `hash_functions.rs`. In a future PR, we should extract and lax-check libcrux-sha3 and then lax-check the _implementation_ of `hash_function.rs` against the `libcrux-sha3` interfaces. The general issue for tracking progress on libcrux-sha3 verification is https://github.com/cryspen/libcrux/issues/395